### DISCUSSION TITLE
如何让 Formily 的数据与 Mobx 增量同步？

### DISCUSSION THREAD

**Discussion by hjkcai** [alibaba/formily#2136] [Q&A 2.x]
我目前在尝试着在一个存量的 mobx 项目中引入 Formily，现有的数据是 mobx 的 observable，现在希望能够直接把现有数据接入 Formily 编辑。但 @formily/reactive 和 mobx 是两套体系（虽然很像），没办法直接简单地把 mobx 的数据放到 Formily 里面，大佬们有没有什么好方法呢？

有个粗暴的做法，就是在 mobx 或者 Formily 任意一方有数据变化时，全量 toJS 到另一方。但这样性能跟不上了（有拖动场景）。

**Comment by janryWang** [comment]
为啥不能直接用@formily/reactive

**Reply by hjkcai** [reply]
存量项目，迁移一遍不容易…😂
